app/components/views/GetStartedPage/ReleaseNotes/Form/Form.module.css

```diff
@@ -84,6 +84,10 @@
   background-image: var(--release-image-v170);
 }
 
+.image.v180 {
+  background-image: var(--release-image-v170);
```

Comment:
Posthumous review, but shouldn't this be 180?

Reply:
Yup, though we have no new art so just going to reuse 1.7.0 until art can be provided.